src/main/java/com/odysseusinc/arachne/datanode/dto/atlas/EstimationListItem.java

```diff
@@ -0,0 +1,73 @@
+package com.odysseusinc.arachne.datanode.dto.atlas;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class EstimationListItem extends BaseAtlasEntity implements Serializable {
```

Comment:
Why not `extends CommonAnalysis`? - `createdBy`, `createdDate`, etc overlap

Reply:
It should be both a `BaseAtlasEntity` to build map between entities in multiple Atlas instances and `CommonAnalysis`. But `Java` is not `Scala`, there no `traits` here.